pkgs/by-name/wi/winchecksec/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pe-parse,
+  uthenticode,
+  openssl,
+  gtest,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "winchecksec";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "trailofbits";
+    repo = pname;
```

```suggestion
    repo = "winchecksec";
```
`pname` isn't reused anywhere else
- https://github.com/NixOS/nixpkgs/issues/277994